OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -492,10 +492,12 @@ addPathPoint(const SimTK::State& s, int aIndex, const PhysicalFrame& frame)
 {
     PathPoint* newPoint = new PathPoint();
     newPoint->setParentFrame(frame);
-    Vec3 location = newPoint->getLocation(s); // 0 by construction
-    // location is now a COPY not a reference so the computed location is LOST unless setLocation is invoked
-    placeNewPathPoint(s, location, aIndex, frame);
-    newPoint->setLocation(location); // 
+    Vec3 newLocation(0.0); 
+    // placeNewPathPoint takes a newLocation reference and changes it 
+	// based on the index in the path (interpolate or extend, in ground frame)
```

Comment:
perhaps something like:

Note: placeNewPathPoint() returns a location by reference. It computes a new location according to the index where the new path point will be inserted along the path (among the other path points).

Reply:
Also, convert tabs to spaces